contents/blog/best-mobile-app-ab-testing-tools.mdx

```diff
@@ -0,0 +1,303 @@
+---
+date: 2023-11-21
+title: The 9 best mobile app A/B testing tools
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/open-source-testing-tools/testinghog.png
+featuredImageType: full
+author:
+  - lior-neu-ner
+category: General
+tags:
+  - Guides
+  - Comparisons
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+Also known as split testing or [multivariate testing](/product-engineers/what-is-multivariate-testing-examples), A/B testing enables you to measure the impact of product changes on your mobile apps.
+
+There are many A/B testing tools out there, but each one is built for a specific target audience in mind. In this guide, we’ll explore these options so that you can decide which is best for you.
+
+## 1. PostHog
+
+![PostHog - best ab testing tools](../images/blog/best-mobile-app-ab-testing-tools/posthog.png)
+
+- **Best for:** Engineering and product teams
+- **Free tier:** <span className="text-green text-lg">✔</span>
+- **Transparent pricing:** <span className="text-green text-lg">✔</span>
+- **Open source:** <span className="text-green text-lg">✔</span>
+- **Get started without talking to sales:** <span className="text-green text-lg">✔</span>
+- **Includes product analytics:** <span className="text-green text-lg">✔</span>
+
+[PostHog](https://posthog.com) (that's us!) is an all-in-one platform built to help engineers create better products. It includes A/B testing **and** a whole bunch more, such as [product analytics](/product-analytics), [session replays](/session-replay), [feature flags](/feature-flags), and [surveys](/survey).
+
+It's designed for product-minded engineers, growth teams, and product managers who need to move fast and iterate based on reliable, actionable insights.
+
+### Which platforms does PostHog support?
+
+<ComparisonTable column1="PostHog">
```

Comment:
I think you should do the same list instead of table for this. The list looks better because you aren't really comparing anything. The "PostHog" title looks out of place.